cl/alerts/templates/disable_docket_alert.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
```

Comment:
For this and the other similar page, could you try following the template here: 

https://www.courtlistener.com/alert/docket/new/?pacer_case_id=178502&court_id=dcd

I think it should apply nicely. The two buttons it has are good, but the second one could be "Manage Alerts" instead.

Reply:
Sure, I've added some screenshots about how it looks now with this template.